src/cdk/explorer/nodes/getCfnDefinition.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+
+/**
+ * @param uniqueIdentifier unique identifier of state machine
+ * @param cdkOutPath cdk.out path
+ * @param stackName name of parent stack 
+ */
+export function getCfnDefinitionForStateMachine(uniqueIdentifier: string, cdkOutPath: string, stackName: string) {
```

Comment:
I think we should rename this function to be `getStateMachineDefinitionFromCfnTemplate` or something along those lines.

Reply:
In general all exported functions should have a function comment describing what they do.